react/features/base/connection/actions.web.js

```diff
@@ -31,21 +34,25 @@ export {
 export function connect() {
     return (dispatch: Dispatch<*>, getState: Function) => {
         const state = getState();
+        const {
+            logCollector
+        } = state['features/base/logger'];
 
         // XXX Lib-jitsi-meet does not accept uppercase letters.
         const room = state['features/base/conference'].room.toLowerCase();
 
         // XXX For web based version we use conference initialization logic
         // from the old app (at the moment of writing).
         return APP.conference.init({ roomName: room }).then(() => {
-            if (APP.logCollector) {
+            if (logCollector) {
                 // Start the LogCollector's periodic "store logs" task
-                APP.logCollector.start();
-                APP.logCollectorStarted = true;
+                logCollector.start();
+
+                dispatch(startLogCollector());
```

Comment:
feature/base/logger implements non-UI logic and must run on both Web and React Native. Keeping in line with splitting your PR in logical, standalone commits, you may refactor Web in one commit but you must have in the PR React Native support.